src/app/examples/stock-market/components/stock-market-container.component.spec.ts

```diff
@@ -181,6 +181,7 @@ describe('StockMarketContainerComponent', () => {
 function createState(stockState: StockMarketState) {
   return {
     examples: {
+      books: {},
```

Comment:
I think this can be left as it was?

Reply:
Yes, you're right. This is a leftover from the first attempt to solve the failing tests. 

In the end it was solved by adjusting setup of the `TestBed` with more granular imports which resulted in NOT initializing `book.effects.ts`.